### PR TITLE
webpage for the OCaml Workshop 2017

### PR DESCRIPTION
**Pull request by gasche**

### CODE REVIEW

File: site/meetings/ocaml/2017/index.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<!-- ((! set title 2016 !)) -->
```
Comment (dsheets) [inline site/meetings/ocaml/2017/index.md]: 2017?

Comment (gasche) [reply site/meetings/ocaml/2017/index.md]: Duh, thanks.

**Comment by pw374** [comment]
It seems nothing points to that page. Can you fix it?

**Comment by gasche** [comment]
@pw374 the [caml-list announcement email](https://sympa.inria.fr/sympa/arc/caml-list/2017-04/msg00002.html) points to this page. But you are right, I'll add a link from the parent page.

**Comment by agarwal** [comment]
The _OCaml 2016_ news item on the front page should be updated to _OCaml 2017_.

**Comment by gasche** [comment]
@agarwal thanks, done in #885.

**Pull request merged**